driver/driver_config_builder.py

```diff
@@ -256,10 +257,8 @@ def _get_cloudwatch_metrics_file(self, db_instance_identifier):
         db_type = get_db_type(db_instance_identifier, self.rds_client)
 
         if "aurora" in db_type:
-            if db_type == "aurora":
-                db_type = "aurora_mysql"
-
-            db_version_breakdown = db_version.split("_")
+            if db_type == "aurora_mysql":
+                db_version_breakdown = db_version.split("_")
```

Comment:
maybe leave this outside? otherwise postgresql cannot find the variable `db_version_breakdown`

Reply:
thanks! Seeing that the tests caught this too. yay tests!